cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/KafkaConnectApi.java

```diff
@@ -133,6 +136,18 @@
      * this returns the list of connector plugins.
      */
     Future<List<ConnectorPlugin>> listConnectorPlugins(String host, int port);
+
+    /**
+     * Make a {@code GET} request to {@code /admin/loggers}.
+     * @param host The host to make the request to.
+     * @param port The port to make the request to.
+     * @param desiredLogging Desired logging.
+     * @return A Future which completes with the result of the request. If the request was successful,
+     * this returns the list of connector loggers.
+     */
+    Future<Void> updateConnectorLoggers(String host, int port, String desiredLogging);
```

Comment:
For this and all the other functions ... are these really _Connector_ loggers? I assumed they are configure for the whole connect and not per connector in which case they would be _Connect_ loggers.